packages/all/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "sideEffects": false,
   "scripts": {
-    "clean": "rm -Rf ./lib/* ./meta/bundlesize/* ./meta/coverage/*",
+    "clean": "rm -Rf ./lib/* ./meta/bundlesize/* ./meta/coverage/* ./.rpt2_cache",
```

Comment:
This is fine for now, but we might want to use `npx rimraf` instead (if any Windows folks complain).

Reply:
I am Windows folk and I can imagine I could be complaining :)

Reply:
Definitely, great point. I've noted this down along with a few other outstanding script related changes, and will get them in place separately. Thanks!